src/messages/layouts/MessageLayoutElement.hpp

```diff
@@ -93,6 +94,23 @@ class ImageWithBackgroundLayoutElement : public ImageLayoutElement
     QColor color_;
 };
 
+class ImageWithCircleBackgroundLayoutElement : public ImageLayoutElement
+{
+public:
+    ImageWithCircleBackgroundLayoutElement(MessageElement &creator,
+                                           ImagePtr image,
+                                           const QSize &imageSize, QColor color,
+                                           int padding);
+
+protected:
+    void paint(QPainter &painter) override;
+
+private:
+    QColor color_;
+    QSize imageSize_;
+    int padding_;
```

Comment:
Guess can be safely const.
```suggestion
    const QColor color_;
    const QSize imageSize_;
    const int padding_;
```

Reply:
Updated in c3af3fe5